filecoin-proofs/src/api/internal.rs

```diff
@@ -158,29 +160,31 @@ fn get_post_params(
     };
 
     Ok(lookup_groth_params(
-        format!("POST[{}]", usize::from(sector_bytes)),
+        format!(
+            "POST[{}]",
+            usize::from(PaddedBytesAmount::from(post_config))
+        ),
         get_params,
     )?)
 }
 
-fn get_zigzag_verifying_key(
-    sector_bytes: PaddedBytesAmount,
-) -> error::Result<Arc<Bls12VerifyingKey>> {
-    let public_params = public_params(sector_bytes);
+fn get_zigzag_verifying_key(porep_config: PoRepConfig) -> error::Result<Arc<Bls12VerifyingKey>> {
+    let public_params = public_params(PaddedBytesAmount::from(porep_config));
 
     let get_verifying_key =
         || ZigZagCompound::verifying_key(&public_params, &ENGINE_PARAMS).map_err(Into::into);
 
     Ok(lookup_verifying_key(
-        format!("ZIGZAG[{}]", usize::from(sector_bytes)),
+        format!(
+            "ZIGZAG[{}]",
+            usize::from(PaddedBytesAmount::from(porep_config))
```

Comment:
Is it true that these keys will need to be updated to include PoRep partitions?

Reply:
Yes, groth parameters will need to be fetched by both sector size and number of partitions.